kahuna/public/js/crop/view.html

```diff
@@ -30,20 +31,25 @@
 
             <div ng:switch-when="conditional"
                  class="cost cost--conditional"
-                 title="{{ctrl.image.data.usageRights.restrictions}}">
+                 title="restricted use: {{ctrl.image.data.usageRights.restrictions}}">
                 <!-- As `conditional` can only be set with usageRights, let's
                      just assume it's here. We might need to revisit this. -->
-                restricted use
-                <span class="ico ico--circled ico--help">?</span>
+                     <i class="material-icons">warning</i><span class="top-bar-item--label-text"> restricted use
+                         <span class="ico ico--circled ico--help">?</span></span>
```

Comment:
Not part of this feature, per se - can we replace this with the [icon](https://www.google.com/design/icons/#ic_help_outline)? We'd be able to remove some css for `ico` then too.